backend/RoomPlanner/room-planner/src/main/java/edu/roomplanner/entity/RoomEntity.java

```diff
@@ -0,0 +1,24 @@
+package edu.roomplanner.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+@Entity
+@Getter
+@Setter
+public class RoomEntity extends UserEntity {
+
+    @Column(name = "room_name")
+    private String roomName;
```

Comment:
room in naming is redundant here, we are already in the RoomEntity class, RoomEntity.getRoomName() is not as elegant as RoomEntity.getName()